### ISSUE TITLE
autopygen: missing module stdlib-list

### ISSUE THREAD

**Issue by thorstenwagner** [galaxyproject/planemo#1432]
I installed the latest planemo via pip. If I try to run it with the --autopygen I run into the following exception:

```
Traceback (most recent call last):
  File "/opt/user_software/miniconda3_envs/_galaxy_/lib/python3.7/site-packages/planemo/cli.py", line 75, in name_to_command
    mod = __import__(mod_name, None, None, ["cli"])
  File "/opt/user_software/miniconda3_envs/_galaxy_/lib/python3.7/site-packages/planemo/commands/cmd_tool_init.py", line 10, in <module>
    from planemo import (
  File "/opt/user_software/miniconda3_envs/_galaxy_/lib/python3.7/site-packages/planemo/tool_builder.py", line 18, in <module>
    from planemo.autopygen.argument_parser_conversion import (
  File "/opt/user_software/miniconda3_envs/_galaxy_/lib/python3.7/site-packages/planemo/autopygen/argument_parser_conversion.py", line 25, in <module>
    from planemo.autopygen.commands.command_utils import (
  File "/opt/user_software/miniconda3_envs/_galaxy_/lib/python3.7/site-packages/planemo/autopygen/commands/command_utils.py", line 16, in <module>
    from planemo.autopygen.param_info import (
  File "/opt/user_software/miniconda3_envs/_galaxy_/lib/python3.7/site-packages/planemo/autopygen/param_info.py", line 11, in <module>
    from planemo.autopygen.source_file_parsing.constants import WARNING_STRING
  File "/opt/user_software/miniconda3_envs/_galaxy_/lib/python3.7/site-packages/planemo/autopygen/source_file_parsing/constants.py", line 7, in <module>
    from stdlib_list import stdlib_list
ModuleNotFoundError: No module named 'stdlib_list'

```

I was able to fix it with:

```
pip install stdlib-list
```

**Comment by thorstenwagner** [comment]
I just saw that `stdlib_list` is in the requirements file. Maybe it must be `stdlib-list`?

**Comment by nsoranzo** [comment]
Thanks for the bug report!

> I just saw that `stdlib_list` is in the requirements file. Maybe it must be `stdlib-list`?

This is not what causes the issue, pip/PyPI treat both as aliases, although the official package name is indeed with a `-`.
Any way, this should be fixed by #1433.
